terraform/README.md

```diff
@@ -0,0 +1,63 @@
+# Terraform module for kubeflow-profiles
+
+This is a Terraform module facilitating the deployment of the kubeflow-profiles charm, using the [Terraform juju provider](https://github.com/juju/terraform-provider-juju/). For more information, refer to the provider [documentation](https://registry.terraform.io/providers/juju/juju/latest/docs) 
+
+## Compatibility
+This terraform module is compatible with charms of version >= ckf-1.9 due to changes in the charm's relations.
```

Comment:
This charm is released to channels with `1.9` track, not ckf-1.9.

Reply:
Good catch Orfea!